### PR TITLE
Awesome Books Milestone :green_circle: 

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
- My partner, @cyrilmuchemi and I have created a book list app with simple HTML and javaScript.
- We have used objects and methods to display books, add books, remove books and store books to local storage

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi team😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please add a descriptive README file
- Please add the correct Linters for the HTML and JavaScript

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Review by codepantha — COMMENTED** [review]
Hi team 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 

You have submitted a project that has no linters and a professional README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```
Comment (keroloussamy) [inline README.md:45]: - [ ] Very good by adding your Readme file, just one thing to make it complete is to add formatting to it to make it looks professional.

You can see this [link](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md), and add the formatting to your Readme file.

**Review by keroloussamy — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**